README.md

```diff
@@ -275,6 +271,142 @@ An example policy would look like;
 }
 ```
```

Comment:
I assume the following sample code relates to IAM policy, but there is no text to describe that.